src/components/RequestGroupComposition/Configuration.vue

```diff
@@ -468,7 +460,15 @@ export default {
       dither: {
         fields: [
           { key: 'ra', label: 'RA offset (arcsec)' },
-          { key: 'dec', label: 'Dec offset (arcsec)' }
+          { key: 'dec', label: 'Dec offset (arcsec)' },
+          { key: 'exposure_time', label: 'Exposure Time (s)' },
+          {
+            key: 'optical_elements',
+            label: 'Optical Elements',
+            formatter: value => {
+              return objAsString(value);
+            }
+          }
```

Comment:
mode or defocus are the only other fields I can think of that could vary, but those are rarely changed so I think leaving them out is fine.